multiple-cursors-core.el

```diff
@@ -98,16 +103,26 @@
        (set-marker ,p nil)
        (set-marker ,s nil))))
 
+(defun mc/cursor-is-bar ()
+  "returns true if the cursor is a bar"
+  (cond ((equalp cursor-type 'bar) t)
+   ((when (listp cursor-type) (equalp (car cursor-type) 'bar)) t)
+   (t nil)))
```

Comment:
Could this be simplified to this?

```
(or (eq cursor-type 'bar)
    (and (listp cursor-type)
         (eq (car cursor-type) 'bar)))
```

(Also, `equalp` should be `cl-equalp`, but I don't think it's needed here anyway)